archaius2-api/build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java'
+
+dependencies {
+    compile     'org.slf4j:slf4j-api:1.6.3'
+    compile     'javax.inject:javax.inject:1'
+    compile     'com.google.code.findbugs:jsr305:3.0.1'
+
+    testCompile project(":archaius2-core")
```

Comment:
Let's move all the tests into core.  Putting this testCompile dependency here actually causes a circular dependency (archaius2-api -> archaius2-core -> archaius2-api) which confuses eclipse.